src/james.data.clss.pas

```diff
@@ -207,13 +207,16 @@ function TDataStream.Save(const FileName: string): IDataStream;
 end;
 
 function TDataStream.AsString: string;
+var
+  Buf: AnsiString;
 begin
   with FStream do
   begin
-    SetLength(Result, Size);
+    SetLength(Buf, Size);
     Position := 0;
-    ReadBuffer(Pointer(Result)^, Size);
+    ReadBuffer(Pointer(Buf)^, Size);
```

Comment:
@nunopicado maybe we can lose information when using a Unicode string.
What do you think about using a `TStringStream`?